examples/scripts/ejb-persistent-clustered-timers.cli

```diff
@@ -0,0 +1,4 @@
+batch
+/subsystem=ejb3/service=timer-service/database-data-store=ejb-timers-ds:add(datasource-jndi-name=java:jboss/datasources/PostgreSQLDS}, database=postgresql, partition=ejb-timers-ds-part)
```

Comment:
There is an extra '}' at the end of the name value.

Reply:
Yep, just caught while testing - but you were quicker! :smiley:

Reply:
I'll put this in draft status while I test the lastest changes (I need to push for Helm to fetch sources)